src/components/image-config/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactElement, ReactNode } from 'react'
+import getValidatedImgChild from './get-validated-image-child'
+import MdxImg from '../mdx-img'
+import s from './image-config.module.css'
+
+/**
+ * Configure extra settings for images via MDX.
+ * Designed to wrap a single markdown image.
+ */
+export default function ImageConfig({
+  /** Hide borders (temporarily configurable while we remove baked-in borders from existing images) */
+  hideBorder = false,
+  /** A caption to display below the image. */
+  caption,
```

Comment:
Should a default caption be set here if it's optional? Just looking at the default props warning below, curious how you see this working in practice. I would imagine that most images at the onset won't have captions.

Reply:
I think the `TEMP_DEV_CAPTION` thing I did made the plan here opaque - if there's no `caption` provided, we'll render a plain `img` tag without a caption instead of a `figure` with a caption. Now that we have the `image-config` component in the main components folder and showing the example in Swingset, I'll remove the `TEMP_DEV_CAPTION` thing and that should make this more clear 🙇‍♂️